pkg/kubernetes/scripts/virtual-machines.js

```diff
@@ -20,14 +20,25 @@
 (function() {
     "use strict";
 
+    /* Expose jQuery and Bootstrap to non-Angular code */
+    var jquery = require('jquery/dist/jquery.js');
```

Comment:
It is needed only for virtual machines because angular has it's own version of `jQuery` and `angular-bootstrap-npm` is used. There is no other non-angular code in `kubernetes` so it is sufficient to leave it only here. I would move it higher when new use case occurs.